.github/workflows/prebuild.yml

```
@@ -21,34 +21,43 @@ jobs:
         with:
           node-version: 14.8.0
 
+      - name: npm install
+        run: npm install
+        env:
+          PREBUILD: true
+
       - name: install Linux dependencies
         run: |
           sudo add-apt-repository ppa:jonathonf/ffmpeg-4
           sudo apt-get update
           sudo apt-get install libavcodec-dev libavformat-dev libavdevice-dev libavfilter-dev libavutil-dev libpostproc-dev libswresample-dev libswscale-dev
         if: ${{ runner.os == 'Linux' }}
 
-      - name: install Mac dependencies
-        run: brew install nasm pkg-config ffmpeg
+      - name: install Mac dependencies (brew)
+        run: brew install nasm pkg-config
+        if: ${{ runner.os == 'macOS'}}
+
+      - name: install Mac dependencies (shared release)
+        run: node ./install_ffmpeg.js
         if: ${{ runner.os == 'macOS'}}
 
       - name: install Windows dependencies
         run: node ./install_ffmpeg.js
         if: ${{ runner.os == 'Windows'}}
 
       - name: prebuild for macOS (x64)
-        run: npx prebuild --runtime napi --arch x64 --prepack copy_mac_libs.js --include-regex '\.(node|dylib)$' --upload ${{ secrets.GITHUB_TOKEN }}
+        run: npx prebuild --runtime napi --arch x64 --include-regex '(\.node|\.dylib|ffmpeg|ffprobe)$' --upload ${{ secrets.GITHUB_TOKEN }}
         if: ${{ runner.os == 'macOS'}}
 
-      - name: prebuild for macOS (arm64)
-        run: npx prebuild --runtime napi --arch arm64 --prepack copy_mac_libs.js --include-regex '\.(node|dylib)$' --upload ${{ secrets.GITHUB_TOKEN }}
-        if: ${{ runner.os == 'macOS'}}
+#      - name: prebuild for macOS (arm64)
+#        run: npx prebuild --runtime napi --arch arm64 --include-regex '(\.node|\.dylib|ffmpeg|ffprobe)$' --upload ${{ secrets.GITHUB_TOKEN }}
+#        if: ${{ runner.os == 'macOS'}}
 
       - name: prebuild for Windows
-        # I can't get the pipe character to work on windows, so to include both .node and .dll files I have this weird regexp
-        run: npx prebuild --runtime napi --include-regex '\.[nd][ol][dl]e?$' --upload ${{ secrets.GITHUB_TOKEN }}
+        run: npx prebuild --runtime napi --include-regex '"(.node|.dll|ffmpeg.exe|ffprobe.exe)$"' --upload ${{ secrets.GITHUB_TOKEN }}
```

were you able to get this to work? I couldn't get `|` to work on windows for some reason… also I think you need to escape the `.`:
```suggestion
        run: npx prebuild --runtime napi --include-regex '"(\.node|\.dll|ffmpeg\.exe|ffprobe\.exe)$"' --upload ${{ secrets.GITHUB_TOKEN }}
```

---

Yes, I got it to work by switching to the weird single and double quotes and switching to git-bash for the shell.